Source/EasyNetQ/Consumer/DefaultConsumerErrorStrategy.cs

```diff
@@ -72,10 +73,9 @@ public virtual AckStrategy HandleConsumerError(ConsumerExecutionContext context,
 
             logger.Error(
                 exception,
-                "Exception thrown by subscription callback, receivedInfo={receivedInfo}, properties={properties}, message={message}",
+                "Exception thrown by subscription callback, receivedInfo={receivedInfo}, properties={properties}",
                 context.ReceivedInfo,
-                context.Properties,
-                Convert.ToBase64String(context.Body)
```

Comment:
maybe we should go with a `ToArray()` here (as you did in `CreateErrorMessage`) because we are in a error part, we don't expect to reach this point so many times, so I think we can throw the performance out of the window to let be able to tell something about the message to the developer getting that error in face. What do you think?

Reply:
> maybe we should go with a `ToArray()` here (as you did in `CreateErrorMessage`) because we are in a error part, we don't expect to reach this point so many times, so I think we can throw the performance out of the window to let be able to tell something about the message to the developer getting that error in face. What do you think?

Agree. Will fix soon.

Reply:
Done